### PR TITLE
Removing Tint Color

### LINKED ISSUE #1248: Weird Elevated Colors when changing theme
## Current behavior

When parsing a light layout using plugin set to Dark color (and vice versa), it will generate weird color results:

image

## Expected behavior

It should follow the known colors.

## How to reproduce it (as minimally and precisely as possible)

Just try to parse an elevated button, for example and then try to change the plugins theme.

## Environment

Related to figma issue [1475](https://github.com/unoplatform/Uno.Figma/issues/1475)

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] UWP
- [ ] MacOS

### PR DESCRIPTION
**Pull request by iurycarlos**
Closes #1248

## PR Type

What kind of change does this PR introduce?

- Bugfix

## Description

Removing deprecated Tint Colors in order to avoid the Theme Changing issue.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Extensions/ControlExtensions.cs
Comment (kazo0) [inline src/library/Uno.Material/Extensions/ControlExtensions.cs:1]: Make sure to remove references to these extensions and resources from the docs as well

File: doc/material-controls-extensions.md [hunk 1]
```diff
@@ -48,90 +48,3 @@ It's control specific and for now, you can only use it with the ToggleButton con
...
 </ToggleButton>
 ```
-
-## Elevation
-
-This feature allows to set the level of elevation to depict on the supported control.
-
-Setting the elevation on [supported controls](#supported-controls) can result in changes to both the shadow and the [surface tint](#surface-tint).
-
-[Material Design Elevation Guidance](https://m3.material.io/styles/elevation/overview)
```
Comment (kazo0) [inline doc/material-controls-extensions.md:58]: We didn't remove the Elevation attached property in the ControlExtensions, right? So we should just remove the mention of surface tint in this section

File: doc/material-getting-started.md [hunk 2]
```diff
@@ -177,7 +177,6 @@ Use this when you want to specify MANUALLY each colors.
                 <Color x:Key="OnSurfaceVariantColor">#A5A0AC</Color>
                 <Color x:Key="SurfaceInverseColor">#313033</Color>
                 <Color x:Key="OnSurfaceInverseColor">#F4EFF4</Color>
-                <Color x:Key="SurfaceTintColor">#5946D2</Color>
```
Comment (kazo0) [inline doc/material-getting-started.md:180]: We shouldn't be removing the SurfaceTintColor, just the extensions surrounding TintedBackground and IsTintEnabled.

We should be following what we did [here](https://github.com/unoplatform/uno.toolkit.ui/pull/987/) to replace the tint in the TabBar

Comment (iurycarlos) [reply doc/material-getting-started.md:180]: Oh!!! Got it!

**Review by kazo0 — APPROVED** [review]

**Comment by kazo0** [comment]
@mergifyio backport release/stable/4.1

**Pull request merged**